Gemfile.lock

```diff
@@ -1,10 +1,10 @@
 GIT
   remote: https://github.com/samvera/hyrax.git
-  revision: d44742f25a628660451736571dda244acd5ffad4
-  branch: main
+  revision: afa8abadebf52c1c5b69168b5d34eca17d7dd6a3
```

Comment:
all changes in this file are the result of changing the docker-compose.yml file and then bundling the app.